src/cpp/rtps/builtin/data/WriterProxyData.cpp

```diff
@@ -65,9 +63,9 @@ WriterProxyData::WriterProxyData(const WriterProxyData& writerInfo)
     , m_typeMaxSerialized(writerInfo.m_typeMaxSerialized)
     , m_topicKind(writerInfo.m_topicKind)
     , persistence_guid_(writerInfo.persistence_guid_)
-    , m_topicDiscoveryKind(writerInfo.m_topicDiscoveryKind)
     , m_type_id(writerInfo.m_type_id)
     , m_type(writerInfo.m_type)
+    , m_type_information(writerInfo.m_type_information)
```

Comment:
These kind of changes were reviewed by @richiprosima , right?

Reply:
Yes